### DISCUSSION TITLE
unable to verify username

### DISCUSSION THREAD

**Discussion by thpietro** [AlexProgrammerDE/balena-minecraft-server#114] [General]
when i try to connect to the server i get the error: "unable to verify username". Is there a way to allow anonymous access to the server?

**Comment by AlexProgrammerDE** [comment]
Are you using cracked minecraft with clients such as `tlauncher`? Or is just your Pi having no internet? In either case you should set online mode to false in server.properties. You can do it as described in the connect to files readme section. Once that's done your server should no longer try to contact mojang auth server when you try to join your server.

**Comment by ghost** [comment]
Im using badlion      Pi internet connection is fine
Op zo 7 mrt. 2021 om 17:53 schreef Alex <notifications@github.com>

**Comment by viedok** [comment]
If your goal is still to allow anonymous access to the server, you should turn online mode off. If you just want the problem resolved, I'd suggest reformatting your SD card, redeploying the project and setting it up again.
Hope this helps.
